serde-generate/src/generate.rs

```diff
@@ -20,29 +20,88 @@ enum Language {
 }
 }
 
+arg_enum! {
+#[derive(Debug, StructOpt, PartialEq, Eq, PartialOrd, Ord)]
+enum Runtime {
+    Serde,
+    Bincode,
+}
+}
+
 #[derive(Debug, StructOpt)]
 #[structopt(
     name = "Serde code generator",
     about = "Generate code for Serde containers"
 )]
 struct Options {
     #[structopt(parse(from_os_str))]
-    input: PathBuf,
+    input: Option<PathBuf>,
```

Comment:
Would this not be a breaking api change?

Reply:
The CLI argument is now optional, so I'm "adding" a feature, no?

Reply:
Anyways, bumping to `0.2.0` looks a bit cleaner.